docs/docs/conditional-page-builds.md

```diff
@@ -6,6 +6,8 @@ If you have a large site, you may be able to improve build times for data update
 
 For more info on the standard build process, please see the [overview of the Gatsby build process](/docs/overview-of-the-gatsby-build-process/).
 
+> ❗ Static queries are currently not stable to use with conditional page builds. Any query result differences will not trigger pages to rebuild.
```

Comment:
I think the issue is that Conditional Page Builds don't account for static queries at the moment. Let's reword this to reflect the same?